pages/style-guide/css.md

```diff
@@ -98,6 +98,7 @@ font-weight:strong;
  - Omit leading zeros in decimal values.
  - Use double quotes instead of single quotes.
  - Always have a semicolon at the end of each declaration.
+ - Avoid the use `!important`.
```

Comment:
Missing word in here. Shouldn't it be `Avoid the use of !important`? Or `Avoid to use !important`?